### PR TITLE
Remove accents from MR name

### PR DESCRIPTION
**Pull request by bwrsandman**
Fix bug in urllib quote when building MR with branch in name

### CODE REVIEW

File: runbot_gitlab/runbot_repo.py [hunk 1]
```diff
@@ -43,13 +44,29 @@
...
+def strip_accents(unicode_string):
+    """Remove accents and greek letters from string
+
+    :param unicode_string: String with possible accents
+    :type unicode_string: unicode
+    :return: String of unicode_string without accents
+    :rtype: unicode
+    """
+    return ''.join(
+        char for char in unicodedata.normalize('NFD', unicode_string)
```
Comment (pedrobaeza) [inline runbot_gitlab/runbot_repo.py:64]: Why not using unidecode that make the change in one call?

Comment (bwrsandman) [inline runbot_gitlab/runbot_repo.py:64]: It's to avoid having them in the url

**Comment by yvaucher** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**